packages/frontend/src/page/components/wallet-position/index.tsx

```diff
@@ -41,10 +42,9 @@ export const WalletPosition = ({
     erc20Name,
 }: WalletPositionProps): ReactElement => {
     const { address: connectedAddress } = useAccount();
-    const { chain } = useNetwork();
     const { data: ensName, isLoading: loadingENSName } = useEnsName({
         address: connectedAddress as Address,
-        enabled: !!(chain && chain.contracts && chain.contracts.ensRegistry),
+        chainId: mainnet.id,
```

Comment:
Why remove the `enabled`?

Reply:
We force the ENS resolution to be performed on mainnet. There's no need to EVER disable the hook if we do that because by definition ENS will always be enabled on mainnet.